src/api/offers.js

```diff
@@ -88,6 +90,9 @@ export default ({offerService, errorService}: { offerService: OfferService, erro
       } else if (offer.expirationDate <= Date.now()) {
         const errorResponse = errorService.createOfferExpiredErrorResponse(token)
         response.status(HttpStatus.GONE).json(errorResponse)
+      } else if (offer.confirmed === false && offer.createdDate < Date.now() - CONFIRMATION_PERIOD) {
```

Comment:
Why do you check for `=== false`? Should `!offer.confirmed` be sufficient?

Reply:
Done